homeassistant/helpers/config_validation.py

```diff
@@ -474,6 +474,60 @@ def validator(config):
     return validator
 
 
+def is_mac_address(value=None, separators=None, allow_lowercase=True,
```

Comment:
This method seems overkill and is not in line with our other config validation. What we do instead is that we pick a format and normalize everything to be that specific format. A method to format mac addresses is already available [here](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/device_registry.py#L41) and could be moved to config_validation.

Also this method does weird things by accepting `value` (unused) and raising `Invalid` (but that is raised outside of the scope of actual validating config)